master/internal/sproto/task_actor.go

```diff
@@ -31,7 +31,8 @@ type (
 	}
 	// TaskContainerStarted contains the information needed by tasks from container started.
 	TaskContainerStarted struct {
-		Addresses []cproto.Address
+		Addresses           []cproto.Address
+		NativeReservationID string
```

Comment:
could you please add a comment what this value is? I assume on agent it's a docker container id (a long hex string). is it a magic k8s-generated pod name on k8s? or also just a hex string?